docs/devguide/newdashboards.asciidoc

```diff
@@ -137,136 +128,56 @@ ES_URL="http://192.168.3.206:9200" make import-dashboards
 
-For visualizing the dashboards of a Beat in Kibana you need to have configured:
+NOTE:: If you want to modify a dashboard that comes with a Beat, it's better to modify a copy of the dashboard as the Beat overwrites the dashboards during the setup phase in order to have the latest version. For duplicating a dashboard, just use the `Clone` button from the top of the page. 
 
-* the Beat index pattern, which specifies how Kibana should display the Beat fields
-* the Beat dashboards, including the dependencies such as visualizations and searches
-
-For the Elastic Beats, the index pattern is available in the GitHub repository of each Beat under
-`etc/kibana/index-pattern` or under the `beats-dashboards` zip archive, available for each Beat release.
 
-For the community Beats, you can easily generate the index-pattern from the `etc/fields.yml` file. For more details
-check the <<generate-index-pattern,generate index pattern>> section.
+Before building your own dashboards or customizing the existing ones, you need to load:
 
-If you would like to build dashboards from scratch for any Elastic Beats, you can start by importing the same version of the index pattern as your Beat:
+* the Beat index pattern, which specifies how Kibana should display the Beat fields
+* the Beat dashboards that you want to customize  
 
-[source,shell]
----------------
-$ scripts/import_dashboards -only-index
----------------
+For the Elastic Beats, the index pattern is available in the Beat package under
+`kibana/*/index-pattern`. The index-pattern is automatically generated from the `fields.yml` file, available in the Beat package. For more details
+check the <<generate-index-pattern,generate index pattern>> section.
 
 After creating your own dashboards in Kibana, you can <<export-dashboards,export the Kibana dashboards>> to a local
 directory, and then <<archive-dashboards,archive the dashboards>> in order to be able to share the dashboards with the community.
 
 [[generate-index-pattern]]
 === Generating the Beat Index Pattern
 
-If you change the fields exported by the Beat, you need to generate a new index pattern for your Beat. Otherwise
-you can just use the index pattern available under the `etc/kibana/index-pattern` directory or in the `beats-dashboards`
-archive for the Elastic Beats.
+The index-pattern defines the format of each field, and it's used by Kibana to know how to display the field.
+If you change the fields exported by the Beat, you need to generate a new index pattern for your Beat. Otherwise, you can just use the index pattern available under the `kibana/*/index-pattern` directory.
 
-The Beat index pattern is generated from the `etc/fields.yml`, where all the fields for a Beat are defined. For each field, besides the `type`, you can configure the
+The Beat index pattern is generated from the `fields.yml`, that contains all the fields generated by the Beat. For each field, besides the `type`, you can configure the
```

Comment:
Users might be confused if we use "generated" to mean a two different things in the same sentence. Maybe it would be better to say: 

```
The Beat index pattern is generated from the `fields.yml`, which contains all
the fields exported by the Beat.
```

Reply:
done